docs/codelabs/building-beautiful-sites/docsite/3-toolbar.md

```diff
@@ -0,0 +1,147 @@
+<!--docs:
+title: "3. Add a toolbar"
+layout: landing
+section: codelab
+path: /codelab/3-toolbar/
+-->
+
+<link rel="stylesheet" href="css/codelab.css" />
+
+# Add a Toolbar
+
+Duration: 6:00
+{: .duration}
+
+Let’s say you bring your beautiful e-commerce skeleton site to a designer on your team. The first thing the designer suggests is to add a [toolbar](https://material.io/guidelines/components/toolbars.html) to better convey the branding and ensure the user always knows where they are.
+
+It is easy to implement the toolbar using MDC-Web’s [mdc-toolbar](https://github.com/material-components/material-components-web/tree/master/packages/mdc-toolbar) component. Use a [fixed toolbar](https://github.com/material-components/material-components-web/tree/master/packages/mdc-toolbar#fixed-toolbars), because it has elevation and floats above the main content.
+
+## Add the Material Icons font
+
+To show the hamburger menu navigation icon in the mocks, add the [Material Icons](http://google.github.io/material-design-icons/#icon-font-for-the-web) font to the web page.
+
+In `index.html`, add the following directly before the `<link>` to `material-components-web.css`
+
+```html
+<link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
+```
+
+You may use the icon font as soon as it loads onto the web page.
+
+## Add the proper mdc-toolbar markup
+
+In `index.html`, replace the `h1` tag with the following markup:
+
+```html
```

Comment:
Have you tried using these codeblocks?

It seems to me these have some weird indentation, not sure if this is something docs introduced.